R/mod_tableGen_utils.R

```diff
@@ -27,7 +27,7 @@ prep_bds <- function(datafile, ADSL) {
     # Bind all the PARAMCD files 
     all_PARAMCD <- bind_rows(PARAMCD, .id = "data_from")  %>% 
       arrange(USUBJID, AVISITN, PARAMCD) %>% 
-      select(USUBJID, AVISITN, AVISIT, PARAMCD, PARAM, AVAL, CHG, data_from)
+      select(USUBJID, AVISITN, AVISIT, dplyr::any_of(c("ATPT", "ATM")), PARAMCD, PARAM, AVAL, CHG, data_from)
```

Comment:
Are we still considering `ATM`? I thought we decided to abandon that since often times it can be a datetime variable with hundreds of values.

Reply:
You are correct. I will remove the ATM variable from here.